src/test/java/uk/gov/ons/ctp/integration/rhsvc/service/DataInCloudServiceIT_Test.java

```diff
@@ -0,0 +1,111 @@
+package uk.gov.ons.ctp.integration.rhsvc.service;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.junit.Before;
+import org.junit.Test;
+import uk.gov.ons.ctp.integration.rhsvc.cloud.CloudService;
+import uk.gov.ons.ctp.integration.rhsvc.cloud.GCSService;
+import uk.gov.ons.ctp.integration.rhsvc.domain.model.Address;
+import uk.gov.ons.ctp.integration.rhsvc.domain.model.CaseContext;
+import uk.gov.ons.ctp.integration.rhsvc.domain.model.UACContext;
+
+import java.io.IOException;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+public class DataInCloudServiceIT_Test {
+
+    CloudService storage;
+    DataInCloud cloud;
+    String caseId;
+    String uacCode;
+    UACContext uac;
+    CaseContext caseContext;
+
+    @Before
+    public void setup(){
+        storage = new GCSService();
+        cloud = new DataInCloudService(storage);
+        caseId = "123456789";
+        uacCode = "abcd-defg-1234";
+
+        String case_type = "(H|HI|C|CI)";
```

Comment:
caseType will be only one of those indicated by the regex ie "H"

Reply:
Done